cli/cmd/plugin/unmanaged-cluster/cluster/kind.go

```diff
@@ -148,6 +156,24 @@ func kindConfigFromClusterConfig(c *config.UnmanagedClusterConfig) ([]byte, erro
 	return rawConfig.Bytes(), nil
 }
 
+func writeKindConfigFile(configBytes []byte, clusterName string) error {
+	configDir := ".config/tanzu/tkg/unmanaged"
```

Comment:
don't we have these conventions defined somewhere? I feel like we need to these paths as well for things like the log file?

Reply:
if not, let's make sure they go into a package somewhere that can easily reference this common destination (e.g. `unmanagedConfigHome`)